lib/types/id.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Number attribute
+ */
+var Attribute = require('../attribute');
+
+module.exports = Attribute.inherit({
+
+    default: null,
+
+    isId: true,
+
+    type: Number,
```

Comment:
yes. it's just default value.
Here is the test for String id
https://github.com/bem-node/promised-models/pull/6/files#diff-26dc00170c88df972fa505a53a2d5cc3R39